tests/state.urls.test.js

```diff
@@ -0,0 +1,7 @@
+import { getMetricURL } from "../src/state/urls";
+
+describe("metric URL replacing", () => {
+  it("works as expected", () => {
+    expect(getMetricURL("foo", "bar.baz")).toEqual("/apps/foo/metrics/bar.baz");
+  });
+});
```

Comment:
A test with a hyphen would be good b/c of the `replace` in the function.

Reply:
Indeed, this test was miswritten! Fixed now